docs/api/sim-card-usage.md

```diff
@@ -195,28 +200,28 @@ SMPP usage for a given **month** in a given **year** can be accessed by using th
-
+Keep in mind that the response is given as a list of usages and not returned as a regular top level JSON object.
 
 ## Call usage for a SIM card
+
 Call usage for a given **month** in a given **year** can be accessed by using the following URL:
 
 `GET /simcards/{icc}/usage/call/{year}/{month}`
 
 **Response structure and types**
 
-| Field           | Type              | Description                                                                                                                  |
-| --------------- | ----------------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| icc             | String            | The unique identifier for the SIM card generating the usage.                                                                 |
-| year            | Number            | Year is given in the ISO 8601 format for example 2019.                                                                            |
-| month           | Number            | Month is represented as a number for example 'april = 4'                                                                     |
-| usageDate       | ISO 8601 DateTime | In UTC time.                                                                                                                 |
-| caller          | String            | E164 MSISDN, e.g. *+4593709603*                                                                                              |
-| called          | String            | E164 MSISDN, e.g. *+4593709603*                                                                                              |
-| duration        | Number            | Duration of the call measured in seconds, the result disregards network connection time. i.e. Only the billed call duration. |
-| fromCountryCode | String            | Alpha-2 code                                                                                                                  |
-| toCountryCode   | String            | Alpha-2 code                                                                                                                  |
-| direction       | [Directions](/general-information/data-types#directions)            |                                 |
+| Field           | Type                                                     | Description                                                                                                                  |
+|-----------------|----------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| icc             | String                                                   | The unique identifier for the SIM card generating the usage.                                                                 |
```

Comment:
Suggestion to add a `<br>` in description text, to create multiline text. With current text length I get a scroll bar at the bottom.